test.ice

```diff
@@ -1,6 +1,10 @@
 
 // This is a file ending in `.ice`.
 
+enum MyEnum {
+    A, B, C
+}
+
 module Hello::There {
     interface Foo {
         void bar();
```

Comment:
It seems like this file is actually using a _JSON_ syntax highlighter, Because this is highlighted correctly:
```suggestion
{      "void": "bar();"}
```
But this file should not be using the JSON syntax highlighter.